app/models/post.rb

```diff
@@ -3,8 +3,11 @@ class Post < ApplicationRecord
   has_many :comments
   has_many :likes
 
-  def update_post_counter
-    # author.posts_counter = author.posts.size
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, numericality: { only_integer: true }
+  validates :comments_counter, comparison: { greater_than_or_equal_to: 0 }
+
+  def update_posts_counter
     author.increment!(:posts_counter)
   end
 
```

Comment:
- [ ] In this Post model be sure to modify your `#five_recent_comments` method to this as well since your tests are falling because of it

  ```
  def five_recent_comments
    comments.order(created_at: :desc).limit(5)
  end
  ```